### ISSUE TITLE
new shortcuts

### ISSUE THREAD

**Issue by kiwi0fruit** [kiwi0fruit/sugartex-completions#2]
Yep.

```json
{
  ".": "⋅",
  "+-": "±",
  "and": "ʌ",
  "array": "⠛",
  "d\\partial": "∂",
  "d\\partial\\bsans": "𝞉",
  "d\\partial\\bf": "𝛛",
  "d\\partial\\bisans": "𝟃",
  "d\\partial\\bi": "𝝏",
  "d\\partial\\it": "𝜕",
  "matrix": "⠋",
  "regular": "˹˺",
  "text": "‹›",
  "v\\mirr": "ʌ",
}
```
closed in v0.2

```js
{
  "P\\para": "¶",
  "P\\mirr": "ꟼ",
  "not": "¬",
  "!": "¬",
  "not\\mirr": "⌙",
  "neg\\mirr": "⌙",
  "!=": "≠",
  "o\\degree": "°",
  "~": "≈",
  "~=": "≃",
  "~=\\": "≅",
  "x": "×",
  "x\\mult": "✕",
  "x\\vec": "⨯",
  ".": "·",
  ".\\oper": "⋅",
  "(x)": "⊗",
  "x\\o": "⊗",
  "o\\x": "⊗",
  "(+)": "⊕",
  "+\\o": "⊕",
  "o\\+": "⊕",
  "and": "ʌ",
  "v\\mirr": "ʌ",
  "v\\rot": "ʌ",
  "v": "∨",
  "or": "∨",
}
```
closed via https://github.com/kiwi0fruit/sugartex-completions/commit/3a186c803d98b11f955f1d8e4cde73a3b7140f02

**Comment by kiwi0fruit** [comment]
closed in v0.2

**Comment by kiwi0fruit** [comment]
`"!": "¬", "not": "¬", "x": ""`, waht about regular math letters?
